hq/app/utils/attempt/Failure.scala

```diff
@@ -59,6 +59,23 @@ object Failure {
     Failure(details, friendlyMessage, 403)
   }
 
+  def cannotGetSecurityGroups(accountId: String): Failure =
+    Failure(
+      s"Cannot retrieve Security Groups for $accountId",
+      s"Cannot retrieve Security Groups for $accountId",
+      500)
+
+ def cannotExposedKeys(accountId: String): Failure =
+    Failure(
+      s"Cannot retrieve exposed IAM keys for $accountId",
+      s"Cannot retrieve exposed IAM keys for $accountId",
+      500)
+def cannotGetCredentialReports(accountId: String): Failure =
+    Failure(
+      s"Cannot retrieve credential report for $accountId",
+      s"Cannot retrieve credential report for $accountId",
+      500)
+
```

Comment:
Maybe all of these could use [awsError](https://github.com/guardian/security-hq/pull/43/commits/0fbfa567d3e22b4deb6dd694d422b8e1b9bb6ebd#diff-18fd0f60f6cd85e31e36fa7086700be8R28) with the relevant string ("Security Groups" / "exposed IAM keys") being passed in? This would remove some duplication. 😄 

Or maybe we could build on awsError to make a serviceError that could be more specific?

Would something like this work:

```
  def cacheServiceError(accountId: String, serviceNameOpt: Option[String]): Failure = {
    val details = serviceNameOpt.fold("Cache service error, unable to retrieve required information for $accountId") { serviceDetails =>
      s"Cache service error, unable to retrieve $serviceDetails for $accountId"
    }
    val friendlyMessage = serviceNameOpt.fold("Unable to retrieve required information for $accountId") { serviceDetails =>
      s"Unable to retrieve $serviceDetails for $accountId"
    }
    Failure(details, friendlyMessage, 500)
  }
```

Or is it not worth it? (YMMV) 🙃 

(copying some comments from the closed PR)